modules/gapi/src/backends/ie/giebackend/giewrapper.cpp

```diff
@@ -25,7 +25,7 @@ using GIEParam = cv::gapi::ie::detail::ParamDesc;
 IE::InputsDataMap giewrap::toInputsDataMap (const IE::ConstInputsDataMap& inputs) {
     IE::InputsDataMap transformed;
     auto convert = [](const std::pair<std::string, IE::InputInfo::CPtr>& p) {
```

Comment:
`convert` is not a good name here, I believe. In fact, you're not converting anything.

I'd rename it /in some further PR/ (now take it to the issue tracker)